### ISSUE TITLE
has no meta file, but it's in an immutable folder. The asset will be ignored.

### ISSUE THREAD

**Issue by ALexanderLonsky** [proyecto26/RestClient#214]
Hello,

1. I created a local package.
2. Added a dependency:
```
"dependencies": {
    "com.proyecto26.restclient": "2.6.2"
  }
```
3. In my main project I imported my local package and added:
```
"scopedRegistries": [
    {
      "name": "OpenUPM",
      "url": "https://package.openupm.com",
      "scopes": [        
        "com.proyecto26.restclient",
        "com.openupm",
        "com.rsg.promise"
      ]
    }
  ] 
```
In my understanding, it's required to add `scopedRegistries` to the main project `manifest` because Unity ignores it in the `package.json` of the local package.
4. Successfully opened the project and imported dependencies.
5. Got these errors:
```
Asset Packages/com.proyecto26.restclient/packages.config has no meta file, but it's in an immutable folder. The asset will be ignored.
Asset Packages/com.proyecto26.restclient/Properties has no meta file, but it's in an immutable folder. The asset will be ignored.
Asset Packages/com.proyecto26.restclient/Properties/AssemblyInfo.cs has no meta file, but it's in an immutable folder. The asset will be ignored.
Asset Packages/com.proyecto26.restclient/Proyecto26.RestClient.csproj has no meta file, but it's in an immutable folder. The asset will be ignored.
....
```

Could you please tell me if the issue is in the way I import RestClient?

Thanks!

**Comment by jdnichollsc** [comment]
Hey mate, thanks for reporting this issue

Did you try using the branch for UPM? https://github.com/proyecto26/RestClient#upm-package

Please let me know!

**Comment by ALexanderLonsky** [comment]
Hey, 
update:
I managed to add it directly to my playground project by the link.

**Comment by imadyTech** [comment]
I just had the same issue after released a pure C# script package to Unity 2021.3.4. Since the package is actually a part of a Visual Studio class library, there was no Meta file and I guess this is the reason for the error. 
I can initiate a Unity project and put all scripts inside the Assets/Scripts folder so the Unity engine will be able to generate meta files automatically. However it will be a little bit weird if the code is used for non-Unity projects.
